### DISCUSSION TITLE
Project API release

### DISCUSSION THREAD

**Discussion by DannyHatcher-Git** [RafaelGB/obsidian-db-folder#601] [Q&A]
I saw in the release of 2.8.3 that there is an experimental link for DBFolder and projects. 

I don't see a view pop up for me to use a database. 
How do I get it to appear as a view option?

**Comment by RafaelGB** [comment]
You only have to install obsidian-projects too and create anew view

**Reply by DannyHatcher-Git** [reply]
That is what I have done, but nothing appears in the view dropdown.

https://user-images.githubusercontent.com/78496825/201692453-fc078dd2-99b2-466a-9a3f-87e9d555e52d.mp4
